### PR TITLE
Add meta-kernel guide

### PR DESCRIPTION
**Pull request by dstansby**
Fixes https://github.com/astrospice/astrospice/issues/9

### CODE REVIEW

**Comment by dstansby** [comment]
@rlaker could you review the new docs I've written for metakernels?

File: docs/guide/coordinates.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+   '2020-12-30 00:00:00.000' '2020-12-31 00:00:00.000'], rsun=695700.0 km, observer=self): (lon, lat, radius) in (deg, deg, km)
+      [(332.12529441,  3.71079513, 1.00114385e+08),
+       (319.00710685,  3.69055562, 9.82750254e+07),
+       (305.93031377,  3.66817727, 9.63683478e+07),
+       ...
+
+Solar system ephemeris
+~~~~~~~~~~~~~~~~~~~~~~
+If a JPL ephemeris is set in astropy, astrospice will automatically use it. If
+not, the 'de440s' ephemeris will be used by deafult. To set a different
```
Comment (rlaker) [inline docs/guide/coordinates.rst:132]: typo for default

File: docs/guide/metakernels.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+Using meta-kernels
+==================
+Meta-kernels contain a collection of other kenels to either download or load into SPICE.
```
Comment (rlaker) [inline docs/guide/metakernels.rst:3]: typo for kernels

**Review by rlaker — APPROVED** [review]
Looks good to me

**Pull request merged**